node/node_cross_link.go

```diff
@@ -103,7 +103,12 @@ func (node *Node) processCrossLinkHeartbeatMessage(msgPayload []byte) error {
 func (node *Node) ProcessCrossLinkMessage(msgPayload []byte) {
 	if node.IsRunningBeaconChain() {
 		pendingCLs, err := node.Blockchain().ReadPendingCrossLinks()
-		if err == nil && len(pendingCLs) >= maxPendingCrossLinkSize {
+		if err != nil {
+			utils.Logger().Debug().
```

Comment:
it would be good to have the error in logs as well
```golang
utils.Logger().Debug().Err(err).
```

Reply:
Added log message.